waspc/examples/todoApp/src/webSocket.ts

```diff
@@ -8,7 +7,7 @@ export const webSocketFn: WebSocketDefinition<
   InterServerEvents
 > = (io, context) => {
   io.on('connection', (socket) => {
-    const username = getFirstProviderUserId(socket.data.user) ?? 'Unknown'
+    const username = socket.data.user?.getFirstProviderUserId() ?? 'Unknown'
```

Comment:
I was using a rule of thumb: if it's a property that they would be able to get just by going through the raw DB entity, I put a getter. If it's something special like getting the first available provider, I used a function.

The options seem to be:
- `user.getIdentities().email`
- `user.identities.email`
- `user.firstProviderId`
- `user.getFirstProviderId()`

The combination felt okay to me, as `user.firstProviderId` feels off since the jobs of getting the first provider id feels dynamic.